test/onnx/parse/roialign_default_test.cpp

```diff
@@ -36,16 +36,31 @@ TEST_CASE(roialign_default_test)
     auto rois = mm->add_parameter("rois", srois);
     auto bi   = mm->add_parameter("batch_ind", sbi);
 
-    // Due to the onnx model using opset 12, the coordinate_transformation_mode should be set to
-    // output_half_pixel
+    // Depending on whether the model was built for Onnx opset 16 or earlier, the default
+    // coordinate_transformation_mode will be different.  These model files had explicit opset given
+    // when they were created.
     auto r = mm->add_instruction(
-        migraphx::make_op("roialign", {{"coordinate_transformation_mode", "output_half_pixel"}}),
+        migraphx::make_op("roialign", {{"coordinate_transformation_mode", "half_pixel"}}),
         x,
         rois,
         bi);
     mm->add_return({r});
-
     auto prog = read_onnx("roialign_default_test.onnx");
-
     EXPECT(p == prog);
+
+    // Opset 12 program
```

Comment:
I suggest making this a separate test so if there is a failure you know which one of the two opsets had the problem

Reply:
Done